src/solvers/refinement/string_refinement.cpp

```diff
@@ -280,7 +284,9 @@ static void substitute_function_applications_in_equations(
   string_constraint_generatort &generator)
 {
   for(auto &eq : equations)
-    eq.rhs() = substitute_function_applications(eq.rhs(), generator);
+  {
+    substitute_function_applications(eq.rhs(), generator);
+  }
```

Comment:
I think the braces are more of a distraction than providing value in this particular case, but that's probably just me :-)